accounts/tests.py

```diff
@@ -14,63 +14,150 @@
 from django.test import TestCase, override_settings
 from django.contrib.auth.models import Group
 import logging
+from django.core.management import call_command
+from django.utils.six import StringIO
+from django.contrib.auth.models import User
```

Comment:
User is being imported twice (Line 8)
(There are actually a lot of unused imports here - but not part of this PR)